decidim-admin/app/models/decidim/admin/concerns/user_extends.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+require "active_support/concern"
+
+module Decidim
+  module Admin
+    # A module that adds functionalities to the Decidim::User model class.
+    module UserExtends
+      extend ActiveSupport::Concern
+
+      included do
+        has_many :participatory_process_user_roles, foreign_key: "decidim_user_id", class_name: Decidim::ParticipatoryProcessUserRole
+        has_many :participatory_processes, through: :participatory_process_user_roles
```

Comment:
I wouldn't give _carte blanche_ to this kind of patching, but in this case I think it's beneficial. @josepjaume if we wanted  to do what you're suggestion we could add another relation with condition. I think having relations in a mode is preferred to just methods, since we can eager load them and benefit from other features associations have.